src/audio/copier/copier.c

```diff
@@ -643,7 +643,7 @@ static int copier_copy(struct comp_dev *dev)
 	switch (dev->ipc_config.type) {
 	case SOF_COMP_HOST:
 		if (!cd->ipc_gtw)
-			return do_endpoint_copy(dev);
```

Comment:
@btian1 if you're going to do this, I think we can go further and remove do_enpoint_copy() altogether isnt it?